vue.config.js

```diff
@@ -1,24 +1,36 @@
 const path = require('path');
+const htmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
   transpileDependencies: ['vuetify'],
-  configureWebpack: {
-    devtool: 'source-map',
-  },
+
+  // `assets` is set in the build script, and in the `.env` file as a default
+  // `dist` is the vue cli default value
   outputDir: process.env.VUE_APP_BUILD_OUTPUT || 'dist',
 
   lintOnSave: false,
 
-  outputDir: 'assets',
   pages: {
     index: {
       entry: 'client/js/main.js',
       template: 'client/public/index.html',
       filename: 'index.html',
+      // https://devtools.vuejs.org/guide/installation.html#standalone
+      ...(process.env.ENABLE_VUE_DEVTOOLS === 'true'
+        ? {
+            vueDevToolsScript: '<script src="http://localhost:8098" data-vue-devtools></script>',
+          }
+        : {}),
```

Comment:
Since the vue cli is interpolating this in to the index.html file, it's webpack under the hook and inherently build time. We can probably look in to moving this stuff to client side eventually, but I'll need to do more research around how/where to do it.